integration/update_field_test.go

```diff
@@ -1119,18 +1119,18 @@ func TestUpdateFieldUnset(t *testing.T) {
 	t.Parallel()
 
 	for name, tc := range map[string]struct {
-		id       string
-		update   bson.D
-		expected bson.D
-		stat     *mongo.UpdateResult
-		err      *mongo.WriteError
-		alt      string
+		id           string
+		update       bson.D
+		expected     bson.D
+		expectedStat *mongo.UpdateResult
+		err          *mongo.WriteError
+		alt          string
 	}{
 		"String": {
 			id:       "string",
 			update:   bson.D{{"$unset", bson.D{{"v", int32(1)}}}},
 			expected: bson.D{{"_id", "string"}},
-			stat: &mongo.UpdateResult{
+			expectedStat: &mongo.UpdateResult{
```

Comment:
Can we move those tests to compat tests? The whole idea is that we don't have to provide expected stats, IDs, etc

Reply:
Added,